src/multi_vector_simulator/cli.py

```diff
@@ -151,5 +160,82 @@ def main(**kwargs):
     return 1
 
 
+def report(pdf=None, path_simulation_output_json=None, path_pdf_report=None):
```

Comment:
Nice to move the code in d00186e5ac4d8877ba8a7375af8549efffa4dc17 into somewhere in the `src` folder - it was always irritating me a bit ;)